src/OrchardCore.Modules/OrchardCore.Notifications/Views/UserNotificationNavbar.cshtml

```diff
@@ -61,7 +61,11 @@
 </script>
 
 <style at="Head">
```

Comment:
Why are we using these instead of having it in a SCSS file?

Reply:
These were already inline. Not sure it's worth creating a new css file (which causes a additional http request) 

Maybe we should just add it to TheAdmin.css and TheTheme.css. I don't recall if we have TheTheme.css already or not.